src/proxy/cache/proxy.js

```diff
@@ -55,16 +67,22 @@ export default _.defaults(
 
         /**
          * Installs the proxy
-         * @param {Object} config
+         * @param {object} config
          */
-        install: function install(config) {
-            var self = this;
-
+        install(config) {
             //install the parent proxy
             qtiServiceProxy.install.call(this);
 
+            const getConfigOption = (name, defaultValue) => {
+                if (config && config.options && config.options.itemCaching) {
+                    defaultValue = parseInt(config.options.itemCaching[name], 10) || defaultValue;
+                }
+                return defaultValue;
+            };
```

Comment:
This doesn't look quite right to me...
- the function has a generic name but only gets numeric values from within `itemCaching` property (quite specific)
- reassignment to `defaultValue` arg when it would make more sense to use another name

If it's intentionally multi-purpose, but limited, maybe explain with a comment.